plugins/services/src/js/structs/Framework.js

```diff
@@ -67,6 +67,9 @@ module.exports = class Framework extends Application {
     // resources of child frameworks won't be aggregated
     return tasks
       .filter(function (task) {
+        return task.state === 'TASK_RUNNING';
```

Comment:
I wondered this too. Is the additional `filter` introduced for readability? With @weblancaster's suggestion we would eliminate an iteration of the array.